packages/utils/src/time.ts

```diff
@@ -148,7 +90,7 @@ export const browserPerformanceTimeOrigin = ((): number | undefined => {
   // performance.timing.navigationStart, which results in poor results in performance data. We only treat time origin
   // data as reliable if they are within a reasonable threshold of the current time.
 
-  const { performance } = WINDOW;
+  const { performance } = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
   if (!performance || !performance.now) {
     _browserPerformanceTimeOriginMode = 'none';
```

Comment:
Ideally what we do with `browserPerformanceTimeOrigin` is to make this `timeOrigin` calculation more generic for browser/node, I generally think it uses a good heuristic to determine `timeOrigin` reliability.